context/app/static/js/components/Search/utils.js

```diff
@@ -52,3 +52,19 @@ export function checkboxFilter(id, name, filter) {
     },
   };
 }
+
+export function resultFieldsToSortOptions(fields) {
+  return fields
+    .map((f) => {
+      const base = {
+        defaultOption: false,
+        label: f.name,
+        field: `${f.id}.keyword`,
+      };
+      return [
+        { ...base, order: 'desc', defaultOption: f.id === 'mapped_last_modified_timestamp' },
```

Comment:
We want the default to be one of the columns in the UI. Our date-time format will sort correctly, because most significant units are first, and single digits are padded.